cookbook/getSortedLocalDateTimes.mjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * getSortedLocalDateTimes will sort an array of zoneless Temporal.DateTime instances by the
+ * corresponding local date and time of day (e.g., for building a conference schedule).
+ *
+ *
+ * @param {Temporal.DateTime[]} dateTimes This is a Date instance
+ * @param {boolean} direction accending or descending order
+ * @returns {Temporal.Absolute} Temporal.Absolute instance
+ */
+function getSortedLocalDateTimes(dateTimes, reverse = false) {
+    let newDateTimes = dateTimes.sort((a, b) => {
+        return a.compare(b);
```

Comment:
should the documentation be more specific about whats on the prototype as opposed to whats a static method then? e.g https://github.com/tc39/proposal-temporal/blob/main/docs/absolute.md

Reply:
Definitely it should, currently the headlines are "Temporal.Absolute.staticMethod" vs "absolute.nonStaticMethod". Do you have any ideas on what would make it clearer?

Reply:
usually i look for ClassName.prototype.method() vs ClassName.method() when reading documentation like MDN.
It would be quite long with "Temporal.Absolute.prototype.nonStaticMethod" though

Reply:
Fair enough! MDN does have the methods listed as e.g. String.prototype.trim on an overview page such as https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String (which is what our documentation is at this point) and as string.trim on the individual pages (which we don't have). Still you're right it'd be long.
Would section headings like `# Constructor`, `# Methods`, and `# Static methods` help though?

Reply:
I think so, we can try that

Reply:
I can't seem to tag you as a reviewer, but how about https://github.com/tc39/proposal-temporal/pull/338 ?

Reply:
@ptomato just seen your PR, yep looks much more clear now, thanks!